conda_smithy/configure_feedstock.py

```diff
@@ -2008,6 +2091,17 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
     if config["test"] is None:
         config["test"] = "all"
 
+    if config["github_actions"]["cancel_in_progress"] is None:
```

Comment:
What is the reasoning of this? Why bind `cancel_in_progress` to whether the runners are self-hosted?

Reply:
To save CI resources by default. Self-hosted runners are usually going to be scarce or costly. If a user wants to not cancel-in-progress, they can explicitly say so, but by default it will cancel previous jobs when self-hosted runners are in use.

Reply:
I would also cancel hosted ones, too. They are scarce (only a limited pool of them) and can also be costly if you are not using your free minutes (or a runner that falls into the free minutes).

I would use the rather simple option of

```
if config["github_actions"]["cancel_in_progress"] is None:
    config["github_actions"]["cancel_in_progress"] = True
```

Probably you can just set the default to `False`?

Reply:
So, the cancel in progress here is configured to cancel all jobs in the same build. For eg: Linux builds will get cancelled if windows builds fail.